package.json

```diff
@@ -27,13 +28,15 @@
     "@prisma/client": "5.7.1",
     "@react-email/components": "0.0.12",
     "@retracedhq/logs-viewer": "2.7.0",
+    "@stripe/stripe-js": "1.54.2",
     "@retracedhq/retraced": "0.7.6",
     "@sentry/nextjs": "7.92.0",
     "@tailwindcss/typography": "0.5.10",
     "bcryptjs": "2.4.3",
     "classnames": "2.5.1",
     "cookies-next": "4.1.0",
     "crypto-js": "4.2.0",
+    "dotenv": "16.3.1",
```

Comment:
Node `v20.6.0` has built-in support for `.env` file. We can use `dotenv` if we want to support older versions of Node.

Reply:
had to convert ts file to js for this to work